scripts/azure-linux_mac.yml

```diff
@@ -47,6 +47,12 @@ steps:
   condition: eq(variables['Agent.OS'], 'Linux')
   displayName: 'Check formatting (linux only)'
 
+- bash: |
```

Comment:
Could move this and the format check above the doxgyen/pip/system headers install steps in order to fail a little faster.